README.md

```diff
@@ -1,30 +1,35 @@
+[Pulumi Registry](https://pulumi.com/registry) is the public index of Pulumi extensions and integrations.
 
-## Authoring a Pulumi Package
+## Adding a Package
 
-We’re always eager to expand that index with new [Pulumi Packages](https://www.pulumi.com/docs/guides/pulumi-packages/). Whether you want to author a new native provider, bridge a provider from the Terraform ecosystem, or create a cloud component with best practices and sensible defaults built in, we’d like to work with you to list it on Pulumi Registry.
-To get started, use our [guide on authoring a Pulumi Package](https://www.pulumi.com/docs/guides/pulumi-packages/how-to-author/) of your own.
+To add a package to the Pulumi Registry, open an issue requesting the addition using the ["New Provider"](https://github.com/pulumi/registry/issues/new?template=new-provider.yml) issue template.
 
-### Publishing a Community Package on the Registry
+For assistance, please reach out on the [Pulumi community slack](https://slack.pulumi.com/) or get in touch with us via this [contact form](https://pulumi.com/contact/?form=registry).
+
+### Preparing a Pulumi Provider Package
 
-#### Community Steps
+We’re always eager to expand that index with new [Pulumi Packages](https://www.pulumi.com/docs/guides/pulumi-packages/). To get started, use our [guide on authoring a Pulumi Package](https://www.pulumi.com/docs/guides/pulumi-packages/how-to-author/) of your own.
 
 To publish a community-maintained package on the Pulumi Registry as a community member:
 
 1. Ensure your provider repo has the following files:
     * `docs/_index.md`, which should contain a summary of the provider's purpose (required) along with a code sample (preferred). This file will render as the index page for your provider ([example](https://www.pulumi.com/registry/packages/aiven/)).
     * `docs/installation-configuration.md`, which should contain links to SDK packages in each language along with instructions for configuring the provider (e.g., required credentials and/or environment variables). This file will render as the Installation & Configuration page for your provider ([example](https://www.pulumi.com/registry/packages/aiven/installation-configuration/)).
 1. Create a release of your provider in GitHub.
-1. Add your package to the [community packages list](./community-packages/package-list.json) via pull request to this repository.
 
-For assistance, please reach out on the [Pulumi community slack](https://slack.pulumi.com/) or get in touch with us via this [contact form](https://pulumi.com/contact/?form=registry).
+### Other Package Types
+
+Terraform/OpenTofu providers which have not yet been bridged can be requested through the ["Any Terraform Provider" wrapper](https://www.pulumi.com/registry/packages/terraform-provider/), though we recommend following the [authoring guide](https://www.pulumi.com/docs/guides/pulumi-packages/how-to-author/) to create published SDKs & refine resource configurations.
```

Comment:
I'm not sure what this means:

> Terraform/OpenTofu providers which have not yet been bridged can be requested through the ["Any Terraform Provider" wrapper](https://www.pulumi.com/registry/packages/terraform-provider/),

I'm also not sure that we do recommend Pulumi users manually bridge providers instead of using Any TF Provider, manually bridging vs any TF bridging have different tradeoffs. Specifically, you trade of of time bridging and maintaining for quality of the resulting provider.

Reply:
> I think we'd recommend the static bridging to anyone who also maintains the upstream provider as it gives them better control and has a better experience with published SDKs. But for users who just want to use a TF provider they don't own, then dynamic makes sense.

I agree with this comment, but that's not what the README says.

Reply:
Suggested rewording is https://github.com/pulumi/registry/pull/7066.